pkg/cli/appregistryserver.go

```diff
@@ -0,0 +1,127 @@
+package cli
```

Comment:
I think this code should still live in the ./cmd folder. Why are we moving it to the ./pkg folder?

Reply:
And this comment goes for all the different sub commands

Reply:
So for example have a folder `cmd/commands` where the code that is now in `pkg/cli` lives? I don't see why that wouldn't work. We would just have to change the `Makefile` a bit I think. I don't have a good argument why it's in `pkg` :)

Reply:
I'm actually suggesting that you can just leave 95% of this code exactly where it is. You can create your operator-registry command as a file in the root of the existing `./cmd/` folder called `main.go` that just imports those sub packages. Then just refactor the existing `main()` functions in each command slightly so that they do exactly what you have done to update them. Right now this PR has a ton of noise that makes it hard to read and I'm just suggesting that we can make this more readable without creating any extraneous files in the `./pkg/` folder.